website/content/en/docs/deployment/pipeline-profiles.md

```diff
@@ -0,0 +1,122 @@
++++
+title = "Pipeline Profiles"
+description = "Use AWS IAM roles for service accounts with Kubeflow Profiles"
+weight = 70
++++
+
+### Pipeline S3 Credential Option IRSA (IAM roles for service accounts)
+When using IRSA (IAM roles for service accounts) as your pipeline S3 credential option any additional profiles created as part of a multi-user deployment besides the preconfigured `kubeflow-user-example-com` will need to be configured with IRSA to access S3. As part of multi-user-isolation, every Pipeline run in Kubeflow Pipelines is ran in an individuals profile user-namespace. A `default-editor` service account exists in every Profile's namespace that is used to run Pipelines as-well as connect to S3. This `default-editor` SA needs to be annotated with an IAM role with sufficient permissions to access your S3 Bucket to run your Pipelines. In the below steps we will be configuring an IAM role with restricted access to a specific S3 Bucket, we recommend following the principle of least privileges. 
+
+>Note: If you choose to run your pipeline with a service account other than the default which is `default-editor`, you must make sure to annotate that service account with an IAM role with sufficient S3 permissions.
+
+### Admin considerations
```

Comment:
not all of them look relevant to the content in this page? also change to heading size according to the page